GOVERNANCE.md

```diff
@@ -0,0 +1,153 @@
+The Video.js project is jointly governed by Brightcove and a Technical Steering Committee (TSC). Responsibilities are divided in the following way:
+
+* Brightcove’s responsibility is to provide logistical and administrative control of the project as well as staffing the day-to-day maintenance of Video.js. Brightcove collaborates with the TSC as a member in the Corporate Shepherd role.
+* The TSC’s responsibility is to discuss, debate, and define the overall technical direction of the project by consensus. In large part, this means representing the perspective of users who are external to Brightcove.
+
+Initial membership invitations to the TSC were given to founders of the project and individuals who had been long-term active contributors to Video.js and who had significant experience with the management of the project.
+
+Membership is expected to evolve over time according to the needs of the project and the contributions of individuals.
+
+### Current TSC Members
+
+* [Ben Clifford](https://github.com/mister-ben) (Brightcove)
+* [Pat O’Neill](https://github.com/misteroneill) (Brightcove, poneill@brightcove.com)
+* [Garrett Singer](https://github.com/gesinger) (LinkedIn)
+* [Steve Heffernan](https://github.com/heff) (Mux)
+* [Matthew McClure](https://github.com/mmcc) (Mux)
+* [Gary Katsevman](https://github.com/gkatsev) (Mux)
+
+### Corporate Shepherd of the TSC
+
+In recognition of its long-term commitment to Video.js, Brightcove holds the special distinction of Corporate Shepherd for Video.js, which involves:
+
+* Brightcove occupies a minimum of one permanent seat on the TSC, reserved for an employee who actively contributes to the project.
+* Brightcove is prominently acknowledged as Corporate Shepherd on the Video.js website.
+* Should the TSC be unable to reach consensus on a topic, Brightcove will settle the question at hand.
+
+## Collaborators
+
+The GitHub repositories in the Video.js organization are maintained by Brightcove and the TSC. Additional Collaborators are added and removed by Brightcove and the TSC on an ongoing basis. All TSC members are considered Collaborators implicitly.
+
+All Collaborators have the following permissions:
+
+* Approve pull requests
+* Merge pull requests
+* Push new tags/releases
+
+Collaborators meet monthly on an optional video call.
+
+### Types of Collaborator
+
+There are two types of Collaborator. A guide for Collaborators of both types is maintained in the Video.js [COLLABORATOR_GUIDE.md](./COLLABORATOR_GUIDE.md). They are distinguished as follows:
+
+#### Community Collaborators
+
+Community Collaborators are individuals from the community who make multiple significant and valuable contributions to Video.js.
+
+These are nominated by existing Collaborators and their nominations are discussed during the monthly TSC meeting.
+
+Community Collaborators who are inactive in the project for a period of 6 months or more are considered to have resigned their status and their access will be revoked.
```

Comment:
I mentioned this in the collaborators meeting, but a year would be my suggestion for this time period. I feel like it may not be uncommon for a developer to take a 6th month hiatus from contributing.